tests/data/claypit/tests/_support/DumbGuy.php

```diff
@@ -4,13 +4,13 @@
  * Inherited Methods
  * @method void wantToTest($text)
  * @method void wantTo($text)
- * @method void execute($callable)
- * @method void expectTo($prediction)
- * @method void expect($prediction)
- * @method void amGoingTo($argumentation)
```

Comment:
This docblock was autogenerated, generator still doesn't generate any comments with `self` return type - https://github.com/Codeception/Codeception/blob/522c7f0f1be5b700ffb8ef8342ba6595f4e16242/src/Codeception/Lib/Generator/Actor.php#L93-L103